src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.cpp

```diff
@@ -42,7 +42,17 @@ void FlybyPoint::SelfInit(){
  */
 void FlybyPoint::Reset(uint64_t CurrentSimNanos)
 {
-
+    if (!this->filterInMsg.isLinked()) {
+        bskLogger.bskLog(BSK_ERROR, ".filterInMsg wasn't connected.");
+    }
+    if (this->flybyModel == cwEquations && !this->asteroidEphemerisInMsg.isLinked()) {
+        bskLogger.bskLog(BSK_ERROR, ".asteroidEphemerisInMsg wasn't connected.");
+    }
+
+    /*! set lastFilterReadTime to 0 */
+    this->lastFilterReadTime = 0;
+    /*! set firstRead to True */
```

Comment:
:large_orange_diamond: Comment can be removed.

Reply:
Done